dvc/tree/gdrive.py

```diff
@@ -564,7 +564,9 @@ def remove(self, path_info):
     def get_file_hash(self, path_info):
         raise NotImplementedError
 
-    def _upload(self, from_file, to_info, name=None, no_progress_bar=False):
+    def _upload(
+        self, from_file, to_info, name=None, no_progress_bar=False, **_kwargs
```

Comment:
Why `_kwargs`? Because of the pylint?

Reply:
Several of the other trees already use `_kwargs` here, so this was keeping that style. May make more sense to just change everything to `kwargs` though.